packages/inter-protocol/src/proposals/econ-behaviors.js

```diff
@@ -40,6 +40,14 @@ const BASIS_POINTS = 10_000n;
  * @property {AdminFacet} psmAdminFacet
  */
 
+/**
+ * @typedef {object} AuctionKit
```

Comment:
the contract is named Auctioneer so (like `auctioneerKit`):
```suggestion
 * @typedef {object} AuctioneerKit
```

Reply:
done